harness/determined/cli/user.py

```diff
@@ -205,6 +244,36 @@ def whoami(parsed_args: Namespace) -> None:
             *AGENT_USER_GROUP_ARGS,
         ]),
         Cmd("whoami", whoami, "print the active user", []),
+        Cmd("edit", edit, "edit user fields", [
+            Arg(
+                "target_user",
+                default=None,
+                help="name of user that should be edited"
+            ),
+            Arg("--display-name", default=None, help="new display name for target_user"),
+            Arg("--username", default=None, help="new username for target_user"),
+            Arg(
+                "--remote",
+                dest="remote",
+                type=lambda s: bool(distutils.util.strtobool(s)),
```

Comment:
This results in an error message like:

```
det user edit: error: argument --remote: invalid <lambda> value: 'idontparse'
```

bools with argparse kind of suck, but this would be better if you passed a function that had a comprehensible name rather than a lambda. Could you put one into `declarative_argparse` and use that?

Reply:
create a new function in `declaritive_argparse.py`

Reply:
Looks great!